.circleci/config.yml

```diff
@@ -84,7 +84,40 @@ jobs:
       - pytest
       - store_test_results:
           path: test-results
-
+  build-deb:
+    docker:
+      - image: debian:9
+    steps:
+      - checkout
+      - run:
+          name: Set envvars
+          command: |
+            echo "export PROJECT_DIR=~/project" >>$BASH_ENV
+            echo "export BUILD_DIR=/tmp/build" >>$BASH_ENV
+            echo "export RESULT_DIR=/tmp/result" >>$BASH_ENV
+            echo "export CLOUDIFY_VERSION=5.1" >>$BASH_ENV
```

Comment:
@tehasdf cannot we generate the version in aumtoicat way to read it from setup.py instead of hard coded here, so every version we need to update that manually

Reply:
oh whoops